src/components/TodoList.js

```diff
@@ -1,24 +1,7 @@
-import React from "react";
-import { useEffect } from "react";
-import { useState } from "react";
 import TodoListItem from "./TodoListItem";
-const TodoList = () => {
-  const [data, setdata] = useState([]);
-  const getData = () => {
-    setdata(JSON.parse(localStorage.getItem("savedTitle")));
-  };
-  const HandleRemove = (id) => {
-    let elements = data;
-    elements = elements.filter((element) => element.id !== id);
-    localStorage.setItem("savedTitle", JSON.stringify(elements));
-    getData();
-  };
-  useEffect(() => {
-    getData();
-  }, []);
+const TodoList = ({ data, HandleRemove }) => {
   return (
```

Comment:
`map` from your TodoListItem file goes to this file

Reply:
Thanks for your review, I will fix it.

Reply:
Great job!